apps/parachain-tracker/src/app/home/hero/hero.component.scss

```
@@ -0,0 +1,124 @@
+.hero-wrapper {
+    position: absolute;
+    width: 100%;
+    height: 616px;
+    left: 0px;
+    top: 0px;
+    display: flex;
+    justify-content: center;
+
+    background: linear-gradient(295.35deg, #A644C9 0%, #E6007A 48.62%);
+}
+
+.hero-txt {
+    float: left;
+}
+
+.hero-svg {
+    /* Logos */
+
+    position: relative;
+    left: 400px;
+    top: -360px;
+    width: 652px;
+    height: 423px;
+    background: url("../../../assets/Logos.svg") no-repeat;
```

Prefer absolute paths, this can be shortened to:
```suggestion
    background: url("/assets/Logos.svg") no-repeat;
```

---

> We shouldn't be committing the sass cache, please remove it. You should turn off automatic sass compilation for this project in your IDE.

I will make sure I add it on .gitignore.

---

Somehow my branch does not work on absolute path. I should get configuration first.